src/elements/loading-indicator-circle/readme.md

```diff
@@ -0,0 +1,26 @@
+The `sbb-loading-indicator-circle` is a component which can be used to indicate progress status
+or an ongoing activity which require some time to complete.
+
+```html
+<sbb-loading-indicator-circle></sbb-loading-indicator-circle>
+```
+
```

Comment:
Do we need to specify that it's possibile to use it within other components (eg.buttons?)